src/peerpod-ctrl/Dockerfile

```diff
@@ -35,6 +35,6 @@ ARG CGO_ENABLED=1
 
 RUN if [ "$CGO_ENABLED" = 1 ] ; then dnf install -y libvirt-libs openssh-clients && dnf clean all; fi
 WORKDIR /
-COPY --from=builder /workspace/peerpod-ctrl/manager .
+COPY --from=builder /work/peerpod-ctrl/manager .
```

Comment:
Any specific reason to change the path here?

Reply:
To fix the "plugin was built with a different version of package XXX" from Peerpod-ctrl log.

The go build env must same for CAA, Peerpod-ctrl, and the cloud provider plugin.